Tc error messages

trevorcummingsCCS opened the pull request:
I have added an additional column error_summary varchar(2000) to the table error_messages

trevorcummingsCCS committed:
Create v0.0.7.sql

trevorcummingsCCS committed:
Update v0.0.7.sql

tommyb82 commented on guided-match/versions/v0.0.7.sql:
```diff
@@ -0,0 +1,5 @@
+Version No Version Description
+v0.0.7     This file is a script to create add an additional column to error_messages */
+
+Alter table error_messages add column error_summary varchar(2000);
```
Suggest we make both `error_message` and `error_summary` columns the same length - and somewhere in between 100 and 2000... there are some messages that are longer than 100 characters in the content spreadsheet - but none are anywhere near 2000.  1000 each for both?

tommyb82 replied:
(and most are identical to each other barring the odd one or two and in those cases the text is very similar)

trevorcummingsCCS replied:
sure no problem.I think it would be better to go longer. Do you want me to amend v0.0.7 or amend v0.0.6 and drop v0.0.7?

 It makes no difference to me.

tommyb82 replied:
Me neither - amend v0.0.6 I guess as none of it is deployed anywhere yet anyway 👍

adrianmilne replied:
> Me neither - amend v0.0.6 I guess as none of it is deployed anywhere yet anyway 👍

It is deployed everywhere as of 5 minutes ago.. :) But no problem - I will do destroys and recreates of everything anyway until we are ready to test properly on PROD (21st Sep)

trevorcummingsCCS committed:
Update v0.0.7.sql

set error_message and summary to same size varchar(1000)

trevorcummingsCCS committed:
Update v0.0.7.sql

trevorcummingsCCS committed:
Update v0.0.6.sql

Amended v0.0.6 as it makes more sense than having a change file to a fie which has not been deployed yet.

trevorcummingsCCS committed:
Update v0.0.6.sql

trevorcummingsCCS committed:
Delete v0.0.7.sql

Changes have been moved back to v0.0.6

trevorcummingsCCS committed:
Update v0.0.6.sql

typo

trevorcummingsCCS commented:
Sorry @adrianmilne it is all good now.

tommyb82 reviewed (approved):
Looks good - minor thing, the version number in the file header comment is `v0.0.4` - didn't notice it the first time round!

trevorcummingsCCS committed:
Update v0.0.6.sql

Updated version number. I think that must have been left there when the file was renamed from v0.0.4.sql to v0.0.6.sql
